lib/http_adapter/angular.js

```diff
@@ -8,8 +8,13 @@
     function AngularAdapter($http, $window) {
         this._$http = $http;
         this._$window = $window;
+        this._timeout = 10000;
     }
 
+    AngularAdapter.prototype.setTimeout = function(timeout) {
```

Comment:
I agree, I could probably generalize this a bit more to allow for arbitrary HTTP options. Something like:

    AngularAdapter.prototype.setHttpOptions({
        timeout: 30000
    });